### PR TITLE
[DDW-754] Update Catalyst links

### PR DESCRIPTION
**Pull request by mchappell**
This PR adds footer links to the voting section of Daedalus to allow users engage more with Project Catalyst

## Todos

- [x] Add footer links

## Screenshots
Footer links (English)
Footer links (Japanese)

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Test

---

## Test Scenarios

**Scenario 1 - Check links while syncing (EN)**
```gherkin
Given that Daedalus is syncing
And the selected language is English
And I am on the voting screen 
When I validate the Catalyst links at the bottom
Then I can see that they go to the following URLs:

- Newsletter: https://bit.ly/3dSZJvx
- Announcements Channel: https://t.me/cardanocatalyst
- Community Chat: https://t.me/ProjectCatalystChat 
- Projects:  https://cardano.ideascale.com/
```
**Scenario 2 - Check links while syncing (JP)**
```gherkin
Given that Daedalus is syncing
And the selected language is Japanese
And I am on the voting screen 
When I validate the Catalyst links at the bottom
Then I can see that they go to the following URLs:

- ニュースレター: https://bit.ly/3dSZJvx
- 告知チャネル: https://t.me/cardanocatalyst
- コミュニティチャット: https://t.me/ProjectCatalystChat 
- プロジェクト:  https://cardano.ideascale.com/
```
**Scenario 3 - Check links when Daedalus is synced (EN)**
```gherkin
Given that Daedalus is synced
And has a Shelley wallet
And the selected language is English
And I am on the voting screen 
When I validate the Catalyst links at the bottom
Then I can see that they go to the following URLs:

- Newsletter: https://bit.ly/3dSZJvx
- Announcements Channel: https://t.me/cardanocatalyst
- Community Chat: https://t.me/ProjectCatalystChat 
- Projects:  https://cardano.ideascale.com/
```
**Scenario 4 - Check links when Daedalus is synced (JP)**
```gherkin
Given that Daedalus is synced
And has a Shelley wallet
And the selected language is Japanese
And I am on the voting screen 
When I validate the Catalyst links at the bottom
Then I can see that they go to the following URLs:

- ニュースレター: https://bit.ly/3dSZJvx
- 告知チャネル: https://t.me/cardanocatalyst
- コミュニティチャット: https://t.me/ProjectCatalystChat 
- プロジェクト:  https://cardano.ideascale.com/
```

**Scenario 5 - Check links when Daedalus is synced (EN, no wallets)**
```gherkin
Given that Daedalus is synced
And doesn't have a Shelley wallet
And the selected language is English
And I am on the voting screen 
When I validate the Catalyst links at the bottom
Then I can see that they go to the following URLs:

- Newsletter: https://bit.ly/3dSZJvx
- Announcements Channel: https://t.me/cardanocatalyst
- Community Chat: https://t.me/ProjectCatalystChat 
- Projects:  https://cardano.ideascale.com/
```
**Scenario 6 - Check links when Daedalus is synced (JP, no wallets)**
```gherkin
Given that Daedalus is synced
And doesn't have a Shelley wallet
And the selected language is Japanese
And I am on the voting screen 
When I validate the Catalyst links at the bottom
Then I can see that they go to the following URLs:

- ニュースレター: https://bit.ly/3dSZJvx
- 告知チャネル: https://t.me/cardanocatalyst
- コミュニティチャット: https://t.me/ProjectCatalystChat 
- プロジェクト:  https://cardano.ideascale.com/
```
---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [ ] In English
  - [ ] In Japanese
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Review by gabriela-ponce — APPROVED** [review]

**Review by dmitrii-gaico — APPROVED** [review]

**Comment by alexander-rukin** [comment]
image
@mchappell stakepool links are too big

**Comment by mchappell** [comment]
> image @mchappell stakepool links are too big

Updated to 14px

File: source/renderer/app/containers/voting/VotingRegistrationPage.js [hunk 1]
```diff
@@ -26,61 +27,68 @@ export default class VotingRegistrationPage extends Component<Props> {
     this.props.actions.router.goToRoute.trigger({ route: ROUTES.WALLETS.ADD });
   };
 
-  render() {
-    const { actions, stores } = this.props;
-    const { app, networkStatus, uiDialogs, wallets, voting, profile } = stores;
-    const { openExternalLink } = app;
+  getInnerContent = (isVotingRegistrationDialogOpen: boolean) => {
+    const { app, networkStatus, wallets, voting } = this.props.stores;
```
Comment (danielmain) [inline source/renderer/app/containers/voting/VotingRegistrationPage.js:31]: if you add profile here? so you dont need the line 58

**Review by danielmain — CHANGES_REQUESTED** [review]
All fine, just a tinny suggestion

**Review by nikolaglumac — APPROVED** [review]

**Review by miorsufianiohk — APPROVED** [review]
LGTM. Tested on 19511. The stake pool link size has also been reduced to 14px. Great job @mchappell

**Review by danielmain — APPROVED** [review]
Perfect :)

**Review by alexander-rukin — APPROVED** [review]

**Pull request merged**